From 30ec65b0fbb28a71c9fa4c95c2ecb3eb4e02ea5d Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Thu, 22 Oct 2026 17:11:44 +0800
Subject: [PATCH 2/4] FROMLIST: fix the behavior of -b

The option only set BUNDLEDIR, not {CA,SM,CS}BUNDLE, so it was
basically useless.

Link: https://github.com/lfs-book/make-ca/pull/41
---
 make-ca | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/make-ca b/make-ca
index d8c6da6..f020686 100644
--- a/make-ca
+++ b/make-ca
@@ -124,6 +124,9 @@ function get_args(){
       -b | --bundledir)
         check_arg $1 $2
         BUNDLEDIR="${2}"
+        CABUNDLE="${BUNDLEDIR}/ca-bundle.crt"
+        SMBUNDLE="${BUNDLEDIR}/email-ca-bundle.crt"
+        CSBUNDLE="${BUNDLEDIR}/objsign-ca-bundle.crt"
         shift 2
       ;;
       -d | --cadir)
-- 
2.56.0

